README.md

```diff
@@ -379,6 +379,12 @@ Our `ClasspathHellDuplicatesCheckRuleTest` detects classes that appear in more t
 Releasing
 ---------
 
+PRE-RELEASE NOTICE: Please before starting any release process, ensure that the you have replaced mysql connector package from mariadb connector/j back to drizzle driver. Files requiring changes are:
+* docker-compose.yml
+* fineract-provider/build.gradle
+* fineract-provider/src/main/resources/META-INF/spring/jdbc.properties
+* kubernetes/fineract-server-deployment.yml
+
```

Comment:
Nah, this will be super confusing... we cannot (should not) have the release be anything different than what we work with during development. Let's figure out a way to avoid this. My proposal is that we simply do not ship any JDBD driver in the Fineract distribution at all, ever.

Instead, why don't we just include a 2-3 lines trivial small fineract.sh script illustrating how to launch `java ...` with both `fineract.jar` and `mariadb-java-client-2.7.0.jar` on the CP? And document that in https://github.com/apache/fineract#instructions-to-build-the-jar-file as well (and also a word about having to add it to Tomcat when using the WAR), as part of this PR. And then we could entirely remove this section for "special instructions" for Releasing. WDYT?

As for actually getting the `mariadb-java-client-2.7.0.jar`, it must be available somewhere under `.gradle/` or so? I'm not just where you can find it... or we could just `wget` (or `curl` - whatever) the MariaDB Connector/J?